resolwe_bio/docker_images/dnaseq/packages-r.txt

```diff
@@ -0,0 +1,4 @@
+argparse@2.0.3
```

Comment:
Install Rsamtools to the version compatible with BioC 3.13 ([2.8.0](https://www.bioconductor.org/packages/3.13/bioc/html/Rsamtools.html)).

We should bump only package versions for those packages that are deeclared (in DESCRIPTION of the package) to not work with R 4.1.1.
`argparse` has no R requirements and should be left as it is. `devtools` works with R > 3.0.2 and should be left as it is, too. IMO, this is fine, other than the missing Rsamtools mentioned above.